charts/neuvector-crd/102.0.6+up2.6.6/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/namespace: cattle-neuvector-system
+  catalog.cattle.io/release-name: neuvector-crd
```

Comment:
Why are we adding neuvector chart here?

Reply:
Removed. Restarted the process from scratch.
Thanks.